ModularTegustation/tegu_items/gadgets/unpowered.dm

```diff
@@ -450,6 +450,14 @@
 	mid_length = 2 SECONDS
 	volume = 20
 
+/obj/item/lobotomizer/suicide_act(mob/living/carbon/user)
+	. = ..()
+	user.visible_message(span_suicide("[user] changes \the [src]'s setting from 'Lobotomize' to 'Decimate'! It looks like [user.p_theyre()] trying to commit suicide!"))
+	var/obj/item/organ/brain/brain = user.getorganslot(ORGAN_SLOT_BRAIN)
+	qdel(brain)
```

Comment:
I dont know why but in my local test
![image](https://github.com/user-attachments/assets/9e22c220-cbfe-4c6c-810f-c750cd994198)
The suicider lives even after having their brain deleted. Strange because there is no runtimes.